apps/crn-frontend/src/network/working-groups/WorkingGroupProfile.tsx

```diff
@@ -7,36 +7,49 @@ import {
   WorkingGroupAbout,
   WorkingGroupPage,
 } from '@asap-hub/react-components';
+import { ResearchOutputPermissionsContext } from '@asap-hub/react-context';
 import { network, useRouteParams } from '@asap-hub/routing';
-import { useWorkingGroupById } from './state';
+import { useCanCreateUpdateResearchOutput, useWorkingGroupById } from './state';
+import WorkingGroupOutputs from './WorkingGroupOutput';
 
 const WorkingGroupProfile: FC = () => {
   const route = network({}).workingGroups({}).workingGroup;
   const { workingGroupId } = useRouteParams(route);
   const { path } = useRouteMatch();
   const workingGroup = useWorkingGroupById(workingGroupId);
+
+  const canCreateUpdate = useCanCreateUpdateResearchOutput(workingGroup);
```

Comment:
Kind of feels weird we have a hook and context for this permission. Couldn't we have just used the hook once in the right place? (( just want an answer no necessarily a change ))

Reply:
To be honest, I've looked at the approach we took in Teams, we could change it if needed.